examples/dynamo_integration/latency_sensitivity_demo/src/latency_sensitivity_demo/compare_sensitivity_perf.py

```
@@ -0,0 +1,535 @@
+}
+
+# ---------------------------------------------------------------------------
+# Trie helpers
+# ---------------------------------------------------------------------------
+
+
+def _collect_sensitivity_map(node: PredictionTrieNode, path: str = "") -> dict[str, int]:
+    """Walk the trie and return {function_name: sensitivity} for leaf nodes."""
+    result: dict[str, int] = {}
+
+    for call_idx, pred in node.predictions_by_call_index.items():
```

Ruff (0.15.2)

[warning] Loop control variable `call_idx` not used within loop body

Rename unused `call_idx` to `_call_idx`

(B007)